core/arch/arm/arm.mk

```diff
@@ -99,11 +102,16 @@ ta-mk-file-export-vars-ta_arm32 += CFG_ARM32_ta_arm32
 ta-mk-file-export-vars-ta_arm32 += ta_arm32-platform-cppflags
 ta-mk-file-export-vars-ta_arm32 += ta_arm32-platform-cflags
 ta-mk-file-export-vars-ta_arm32 += ta_arm32-platform-aflags
+
+ta_arm32-set-cross-1 = 'CROSS_COMPILE32 ?= $$(CROSS_COMPILE)'
```

Comment:
Yes. For some reason it took me some time, couldn't handle the line breaks properly. In the end I adopted the solution used in `mk/checkconf.mk`, i.e., use a special token: `_nl_` and replace it at the last moment. Will push an update shortly.